<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UVPSMET" failures="1" tests="6" name="mytest" time="41.303" errors="0" timestamp="2025-05-23T11:53:26 IST">
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDataProvider" time="2.857"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDataProvider" time="2.698"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDataProvider" time="3.255"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDataProvider" time="11.569"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDataProvider" time="2.609">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDataDrivenTest.verify_loginDataProvider(TC003_LoginDataDrivenTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDataProvider -->
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDataProvider" time="11.628"/>
</testsuite> <!-- mytest -->
